bin/rtlcss.js

```diff
@@ -9,7 +9,7 @@ var postcss = require('postcss')
 var rtlcss = require('../lib/rtlcss')
 var configLoader = require('../lib/config-loader')
 
-var input, output, directory, ext, config, currentErrorcode, arg
+var input, output, directory, ext, config, currentErrorcode, arg, exclude
```

Comment:
exclude to be grouped with other options variables.
i.e. `var input, output, directory, ext, config, exclude, currentErrorcode, arg`